<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Login</class>
 <widget class="QDialog" name="Login">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>388</width>
    <height>429</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>388</width>
    <height>429</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>388</width>
    <height>429</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
#UserName_Edit{
	border-top-left-radius:10px;
	border-top-right-radius:10px;
	border-bottom-right-radius:10px;
	border-bottom-left-radius:10px;	
	border:1px solid rgb(209,209,209);
}
#PassWord_Edit{
	border-top-left-radius:10px;
	border-top-right-radius:10px;
	border-bottom-right-radius:10px;
	border-bottom-left-radius:10px;	
	border:1px solid rgb(209,209,209);
}
#UserName_Edit:hover{
	
	border:1px solid rgb(21,131,221)
	
}
#PassWord_Edit:hover{
	
	border:1px solid rgb(21,131,221)
	
}

/*登录按钮样式*/
#Login_Button{
	border-radius:10px;
	border:none;
	background-color: rgb(37,88,184);
	color:#fff;
}

/*登录按钮鼠标悬停样式*/
#Login_Button:hover{
	background-color:rgb(57,159,227);
}
/*游客进入按钮样式*/
#Visitor_Button{
	border-radius:4px;
	border:none;
	background-color: none;
	color:rgb(51,151,224);
}

/*游客进入按钮鼠标悬停样式*/
#Visitor_Button:hover{
	background-color:none;
	color:rgb(43,58,141);
}
/*注册按钮样式*/
#Register_Button{
	border-radius:4px;
	border:none;
	background-color: none;
	color:rgb(51,151,224);
}

/*注册按钮鼠标悬停样式*/
#Register_Button:hover{
	background-color:none;
	color:rgb(43,58,141);
}



/*关闭按钮样式*/
#Close_btn{
	/*border-radius:9px;*/
	border:none;
	background-color: none;
	color:rgb(155,155,155);
}

/*关闭按钮鼠标悬停样式*/
#Close_btn:hover{
	background-color:none;
	color:rgb(0,0,0);
}


/*底部*/
#Bottom{
	background-color:rgb(246,246,246)
}</string>
  </property>
  <widget class="QLineEdit" name="UserName_Edit">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>230</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="PassWord_Edit">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>280</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
  </widget>
  <widget class="QPushButton" name="Register_Button">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>230</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>注册</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Visitor_Button">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>280</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>游客进入</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Login_Button">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>350</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 18pt &quot;Arial&quot;;
font: 12pt &quot;Arial&quot;;</string>
   </property>
   <property name="text">
    <string>登录</string>
   </property>
  </widget>
  <widget class="QLabel" name="UserName_Label">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>230</y>
     <width>21</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/images/User_Black.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="PassWord_Label">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>280</y>
     <width>21</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/images/Password_Black.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="Background_Label">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>-30</y>
     <width>271</width>
     <height>271</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/images/Top.jpeg);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="Bottom">
   <property name="geometry">
    <rect>
     <x>-20</x>
     <y>-50</y>
     <width>461</width>
     <height>551</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>Bottom</zorder>
  <zorder>Background_Label</zorder>
  <zorder>UserName_Edit</zorder>
  <zorder>PassWord_Edit</zorder>
  <zorder>Register_Button</zorder>
  <zorder>Visitor_Button</zorder>
  <zorder>Login_Button</zorder>
  <zorder>UserName_Label</zorder>
  <zorder>PassWord_Label</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
